composer/content.md

```diff
@@ -75,6 +77,16 @@ Sometimes dependencies or Composer [scripts](https://getcomposer.org/doc/article
 
 -	Create your own image (possibly by extending `FROM composer`).
 
+**Note:** Docker introduced [multi-stage](https://docs.docker.com/engine/userguide/eng-image/multistage-build/) builds in 17.05:
+
+-	Create your own image, and copy Composer from the official image into it:
+
+	```
+	COPY --from=composer:1.5 /usr/bin/composer /usr/bin/composer
```

Comment:
`--from` references a stage AFAIK, so this needs to be either a number, or a stage name, right?

Reply:
Indeed, see https://docs.docker.com/engine/reference/builder/#copy:
> Optionally COPY accepts a flag --from=<name|index> that can be used to set the source location to a previous build stage (created with FROM .. AS <name>) that will be used instead of a build context sent by the user. The flag also accepts a numeric index assigned for all previous build stages started with FROM instruction. In case a build stage with a specified name can’t be found an image with the same name is attempted to be used instead.

Reply:
Ok, my reading comprehension skills are low -- it literally does copy from another image, which is terrifying for official images (for similar reasons to what's discussed in https://github.com/docker-library/official-images/issues/3383#issuecomment-329259161 but with the side benefit of being even harder to scrape properly), but in the context of the docs that's totally fine. :innocent: :+1:

Reply:
Admittedly, that is also how I understood it the first time I read it. That last sentence is a bit tricky.